psk_fitness/psk_fitness/Middleware/LoggingMiddleware.cs

```diff
@@ -0,0 +1,67 @@
+namespace psk_fitness.Middleware;
+
+using System.Diagnostics;
+using System.Security.Claims;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using psk_fitness.Properties;
+
+public class LoggingMiddleware
+{
+    public readonly RequestDelegate _next;
+    public readonly IOptionsMonitor<CustomLoggingOptions> _options;
+
+    public LoggingMiddleware(RequestDelegate next, IOptionsMonitor<CustomLoggingOptions> options)
+    {
+        _next = next;
+        _options = options;
+    }
+
+    public async Task InvokeAsync(HttpContext context)
+    {
+        if (_options.CurrentValue.LoggingEnabled) {
+
+            // Get user information
```

Comment:
What is this comment

Reply:
That this section of code is about getting user information from HttpContext

Reply:
hat this section of code is about getting user information from HttpContext

Reply:
ahhh, I see